Gemfile

```diff
@@ -15,6 +15,7 @@ gem 'newrelic_rpm', '~> 8.0'
 gem 'pg'
 gem 'pry-rails'
 gem 'puma'
+gem 'bootsnap', '~> 1.0', require: false
```

Comment:
should we add this to a `development` only group?

Reply:
I'm comfortable running it in production. I've tested it in my sandbox and it's worked ok so far on the IDP side in prod.

Reply:
oh I didn't realize the IDP ran it in prod too!

Reply:
Yep, as of https://github.com/18F/identity-idp/pull/5444!

Reply:
I love how I even approved that PR and had comments 🤦